### PR TITLE
Index test html

### PR DESCRIPTION
**Pull request by MohamedOmarii**

### CODE REVIEW

File: Test/tests.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+  </head>
+  <body>
+    <form action="index.html" method="post">
+      <input type="text" name="searchMovie">
+      <button type="submit" name="submit">serach</button>
+    </form>
+
+    <div class="infoMovie"></div>
+    <div class="giphyMovie"></div>
```
Comment (mattlub) [inline Test/tests.html]: you need to import the QUnit scripts. See http://qunitjs.com/ 'Getting Started' section.

File: Test/tests.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+    <script src="tests.js" type="text/javascript"></script>
+    <script src="fixtures.js" type="text/javascript"></script>
```
Comment (mattlub) [inline Test/tests.html]: you should include our other js files, e.g. render.js

**Review by mattlub — CHANGES_REQUESTED** [review]

File: Test/tests.html [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  <head>
+    <meta charset="utf-8">
+    <title>Kan-Elbash WEB</title>
+    <link rel="stylesheet" href="style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+  </head>
+  <body>
+    <form action="index.html" method="post">
+      <input type="text" name="searchMovie">
+      <button type="submit" name="submit">serach</button>
```
Comment (mattlub) [inline Test/tests.html]: typo here- 'serach' instead of 'search'

File: Test/tests.html [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Kan-Elbash WEB</title>
+    <link rel="stylesheet" href="style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+  </head>
+  <body>
+    <form action="index.html" method="post">
+      <input type="text" name="searchMovie">
```
Comment (mattlub) [inline Test/tests.html]: maybe add 'placeholder' attribute to input tag?

**Review by mattlub — CHANGES_REQUESTED** [review]

File: Test/tests.html (hunk 2 above)
Comment (MohamedOmarii) [reply Test/tests.html]: yeah thank you @mattlub

File: Test/tests.html (hunk 1 above)
Comment (MohamedOmarii) [reply Test/tests.html]: @mattlub good job

**Review by mattlub — APPROVED** [review]
looks good to me

**Pull request merged**